test/extended/pods/priorityclasses.go

```diff
@@ -49,6 +51,11 @@ var _ = Describe("[sig-arch] Managed cluster should", func() {
 			if !hasPrefixSet(pod.Namespace, namespacePrefixes) {
 				continue
 			}
+			// exclude -debug pods, possibly started with "oc debug"
+			if strings.HasSuffix(pod.Name, "-debug") {
+				e2e.Logf("Ignoring debug pod %s/%s", pod.Namespace, pod.Name)
+				continue
+			}
```

Comment:
I'm not sure what the best way to handle this is -- ignore any pod that ends in  -debug? Only allow it in an openshift-must-gather namespace? Something else?